server/api/routes/users.py

```diff
@@ -32,8 +32,8 @@ def login():
         }, 404
     if user.password != req['password']:
         return {
-            'email': 'An account was found but the password is incorrect.',
-            'password': ''
+            'email': '',
+            'password': 'An account was found but the password is incorrect.'
```

Comment:
This message can honestly be something as short as "Password is incorrect". Just thinking of how other websites do it. They don't usually mention that an account was found.